.github/workflows/main.yml

```
@@ -28,14 +29,15 @@ jobs:
              UPSTREAM_WORKSPACE: 'dependencies.rosinstall',
```

```suggestion
             TARGET_WORKSPACE: '. dependencies.rosinstall',
```

The dot means target repository.

---

this puts all packages into the targer workspace, but it will build and run all tests as well.